### PR TITLE
fix: update paypal config after changes to amount, stup out the same …

### PR DESCRIPTION
**Pull request by mcstover**
…for google and apple

### CODE REVIEW

File: src/components/Payment/BraintreeDropInInterface.vue [hunk 1]
```diff
@@ -83,6 +83,37 @@ export default {
...
+			};
+		}
+	},
+	watch: {
+		amount(next, prev) {
+			if (next !== prev) {
+				this.btDropinInstance.updateConfiguration('paypal', 'amount', this.formattedAmount);
+				// eslint-disable-next-line max-len
+				// this.btDropinInstance.updateConfiguration('googlePay', 'transactionInfo', this.googleTransactionInfo);
+				// this.btDropinInstance.updateConfiguration('applePay', 'paymentRequest', this.applePaymentRequest);
```
Comment (mcstover) [inline src/components/Payment/BraintreeDropInInterface.vue:113]: Uncommenting these locally didn't result in any errors but I'm unsure of how we'll test them. @eddieferrer did you help with this originally? Do you know if we can verify them in DEV?

File: src/components/Payment/BraintreeDropInInterface.vue [hunk 2]
```diff
@@ -83,6 +83,37 @@ export default {
...
+				totalPriceStatus: 'FINAL',
+				totalPrice: this.formattedAmount,
+				currencyCode: 'USD',
+			};
+		}
+	},
+	watch: {
+		amount(next, prev) {
+			if (next !== prev) {
+				this.btDropinInstance.updateConfiguration('paypal', 'amount', this.formattedAmount);
```
Comment (mcstover) [inline src/components/Payment/BraintreeDropInInterface.vue:110]: I verified that with this config update sequence in place the amount value that Paypal is tracking is indeed updated after a donation change resulting in a basket total amount change. Once logged into the Paypal popup you can see the amount notated under the credit option, close the popup adjust the basket then re-open Paypal and see the new amount reflected.

**Review by dyersituations — APPROVED** [review]

File: src/components/Payment/BraintreeDropInInterface.vue (hunk 1 above)
Comment (mcstover) [reply src/components/Payment/BraintreeDropInInterface.vue]: Ok, it looks like we can at least make sure both Google and Apple initialize correctly once we're in dev. So, I've uncommented these too.

**Review by emuvente — APPROVED** [review]
Awesome!

File: src/components/Payment/BraintreeDropInInterface.vue (hunk 1 above)
Comment (eddieferrer) [reply src/components/Payment/BraintreeDropInInterface.vue]: @mcstover instructions for testing the complete flow for Google pay and applePay can be found here: https://kiva.atlassian.net/wiki/x/3QA9Kg

**Review by eddieferrer — APPROVED** [review]

**Pull request merged**